keps/sig-scheduling/3902-decoupled-taint-manager/kep.yaml

```diff
@@ -47,4 +48,5 @@ disable-supported: true
 
 # The following PRR answers are required at beta release
 metrics:
-  - TBD
+  - taint_eviction_controller_pod_evictions_total
```

Comment:
The name of the metric was changed in the implementation: https://github.com/kubernetes/kubernetes/pull/119208/files#r1372199155. Can you please update it in the whole KEP?

Reply:
updated.